### DISCUSSION TITLE
4.3 声明式编程

### DISCUSSION THREAD

**Discussion by giscus** [csfive/composing-programs-zh#97] [Announcements]
# 4.3 声明式编程

SICP Python 中文翻译

https://composingprograms.netlify.app/4/3

**Comment by ttyy33** [comment]
“存储在数据库中的每个值都称为记录（record）”这里的翻译错了，应该是“存储在数据库中的每一行数据称为一个记录”

**Comment by yama-lei** [comment]
“追逐猫咪的狗也追逐了鸟”。

原文为 the dog that chased the cat that chased the bird also chased the bird

**Comment by haoaaa-111** [comment]
这个具体实现的py代码我看了一天没有看顺

**Comment by RuiYu2021** [comment]
4.3.1节的第一个例子，下载SqLite后跑不通，需要把双引号改为单引号，才表示这是一个字符串：select 38 as latitude, 122 as longitude, 'Berkeley' as name;
